### PR TITLE
svk_quantify bug fixes

### PR DESCRIPTION
**Pull request by beckolson**
Fix for issues with svk_quantify that caused it to scale output data and not respect the -b flag for limiting calculations to the volume selection. A flag was added so that the user could request scaling instead of casting which is now the default behavior. All tests passed.

### CODE REVIEW

**Comment by beckolson** [comment]
This is a fix the following ticket:
https://sourceforge.net/p/sivic/tickets/221/

File: applications/cmd_line/src/svk_quantify.cc [hunk 1]
```diff
@@ -279,12 +295,9 @@ int main (int argc, char** argv)
             quantMets->LimitToSelectedVolume();
         }    
         quantMets->SetInput( reader->GetOutput() ); 
-        quantMets->LimitToSelectedVolume();
```
Comment (jccrane) [inline applications/cmd_line/src/svk_quantify.cc]: Hi,

Shouldn't this be conditional based on the -b flag, rather than deleted?

-jason

Comment (beckolson) [inline applications/cmd_line/src/svk_quantify.cc]: There is a conditional statement a few lines above. This line was overriding the conditional state set by the -b flag.

**Comment by jccrane** [comment]
I'm guessing the tests passed.

**Pull request merged**